govcd/feture_flag.go

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2025 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"fmt"
+
+	"github.com/vmware/go-vcloud-director/v3/types/v56"
+)
+
+const labelFeatureFlag = "Org VDC Network Segment Profile"
+
+func (vcdClient *VCDClient) UpdateFeatureFlag(entityConfig *types.FeatureFlag) (*types.FeatureFlag, error) {
```

Comment:
Would be nice to have a comment explaining a bit more about the feature flag, as it is kind of perpendicular to other things we have.

Reply:
Added